Update RabbitMQ docs to core V8 (#5230)
Particular/docs.particular.net: transports/upgrades/rabbitmq-6to7.md

## Timeout manager

The [timeout manager is removed from core](/nservicebus/upgrades/7to8/#timeout-manager-removed) which makes timeout manager backwards compatibility mode obsolete. If backwards compatibility mode was enabled these APIs must be removed.

## Configuring RabbitMQ Transport

> To use Azure Service Bus Transport for NServiceBus, create a new instance `AzureServiceBusTransport` and pass it to `EndpointConfiguration.UseTransport`.

Reviewer: ```suggestion
To use the RabbitMQ transport for NServiceBus, create a new instance of the `RabbitMQTransport` and pass it to `EndpointConfiguration.UseTransport`.
```

```

The mandatory configuration settings, the topology and the connection string, are now required to construct the instance of the transport definition class.

## Certificate path and passphrase

Certificate file path and passphrase can now be only passed via the connection string. When configuring secure connection via API the only option is to pass an instance of the `X505Certificate2` class. This instance can be constructed using path and passphrase.

Instead of this code:

snippet: 6to7certificatepath6

Use this:

snippet: 6to7certificatepath7

## Prefetch count

The two prefetch count settings have been replaced with a single one that uses a callback. Instead of either of these APIs:

snippet: 6to7prefetchcount6

Use one of these:

snippet: 6to7prefetchcount7

## Disabling the durable exchanges and queues

Disabling the durable exchanges and queues has been moved to the constructor of the topology classes, `ConventionalRoutingTopology` and `DirectRoutingTopology`. In order to set the value of that parameter use the variant of the `RabbitMQTransport` constructor that accepts an instance of the topology.

## Configuration Options

> The RabbitMQ Transport configuration options that have not been changed have been moved to the `RabbitMQTransport` class. See the following table for further information:

Reviewer: ```suggestion
The RabbitMQ transport configuration options that have not been changed have been moved to the `RabbitMQTransport` class. See the following table for further information:
```
